Sources/Specs/dictionaryManagerSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Dictionary
+import Swiftest
+
+class DictionaryManagerSpec: Swiftest.Spec {
+	let spec = describe("dictionary logic") {
+		var dictionaryManager: DictionaryManager!
+		before() {
+			dictionaryManager = DictionaryManager()
+		}
+
+		describe("#getRandomWord") {
+			var gameWords: [String]!
+			var word: String!
+			before() {
+				gameWords = words
```

Comment:
this assignment is weird, maybe words should remain internal, and `gameWords` could be assigned as `dictionaryManager.words` instead